README.md

```diff
@@ -1,2 +1,110 @@
-# terraform-google-log-export
-A Cloud Foundation Toolkit Module: Opinionated Google Cloud Platform project creation and configuration with Shared VPC, IAM, APIs, etc.
+# Terraform Logsink Module
+
+## Requirements
+### Terraform plugins
+- [Terraform](https://www.terraform.io/downloads.html) 0.10.x
+- [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) plugin v1.8.0
+
+### Configure a Service Account
+In order to execute this module you must have a Service Account with the following:
+
+#### Roles
+The service account with the following roles:
+- roles/logging.configWriter on the logsink project (to create logsinks)

```

Comment:
I would suggest reorganizing this section to list by:
- Roles required for all log sinks (configWriter and serviceUsage)
- Roles required for bigquery
- Roles required for storage
- Roles required for PubSub

Reply:
Got it